README.md

```diff
@@ -37,7 +43,7 @@ Edit a the local .env file by adding the appropriate config values. How to get T
     2. `heroku config:set api_secret="your_consumer_secret_key_goes_here"`
     3. `heroku config:set access_token="your_access_token_goes_here"`
     4. `heroku config:set access_token_secret="your_access_token_secret_goes_here"`
-5. Configure pet datasource–you have two options (below). Search on Petfinder for your shelter and use that if you find it, otherwise look on PetHarbor. 
+5. Configure pet datasource–you have two options (below). Search on Petfinder for your shelter and use that if you find it, otherwise look on PetHarbor. If you want to pull data from multiple shelters (choosing at random), make the shelter id variable a comma-separated list.
```

Comment:
That happens before the API request is sent. See [here](https://github.com/codeforamerica/CutePets/blob/master/lib/cute_pets/pet_fetcher.rb#L128).